backend/typescript/services/implementations/camperService.ts

```diff
@@ -148,7 +138,9 @@ class CamperService implements ICamperService {
     let waitlistedCamperDtos: Array<WaitlistedCamperDTO> = [];
 
     try {
-      const existingCamp: CampSession | null = await MgCamp.findById(campId)
+      const existingCamp: CampSession | null = await MgCampSession.findById(
```

Comment:
ik this wasn't a part of your code but could you rename all of the variables here containing 'camp' to 'campSession'? to make things more readable

Reply:
yes ma'am can do